### PR TITLE
Add link to figure

### PR DESCRIPTION
**Pull request by NatasaKralj**
Figure shortcode currently sets any image src as a link, to be able to open the image in a new tab/browser on click. This does not play nice with any link that follows in `[{{< figure src="someImage.png" >}}](https:alinktosomepage)` format. A quick workaround is to insert link into figure, like this `[{{< figure src="someImage.png" link="https:alinktosomepage" >}}](https:alinktosomepage)`.

### CODE REVIEW

File: content/en/docs/howto/general/install.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ toc-level: "2"
 tags: ["studio pro", "install", "install studio pro", "download"]
 ---
 
-[{{< figure src="/attachments/howto/general/install/download.png" >}}](https://marketplace.mendix.com/link/studiopro/)
+[{{< figure src="/attachments/howto/general/install/download.png" link="https://marketplace.mendix.com/link/studiopro/" >}}](https://marketplace.mendix.com/link/studiopro/)
```
Comment (MarkvanMents) [inline content/en/docs/howto/general/install.md]: Can just be:
```suggestion
{{< figure src="/attachments/howto/general/install/download.png" link="https://marketplace.mendix.com/link/studiopro/" >}}
```
Any reason you want to keep the old method as well?

**Review by MarkvanMents — CHANGES_REQUESTED** [review]
Can we immediately simplify the links, rather than leaving the legacy link as well?

File: content/en/docs/howto/general/install.md (hunk 1 above)
Comment (NatasaKralj) [reply content/en/docs/howto/general/install.md]: The suggestion is now applied to every edited link. No need for this to be a temporary fix, getting rid of the markdown link allows this to be the permanent solution. Thank you @MarkvanMents

**Review by MarkvanMents — APPROVED** [review]
Looks great.
Go for it.

File: content/en/docs/refguide7/desktop-modeler/application-logic/microflows/_index.md [hunk 2]
```diff
@@ -53,49 +53,49 @@ Events represent start and end points of a microflow and special operations in a
...
+| {{< figure src="/attachments/refguide7/desktop-modeler/application-logic/microflows/917902.png" link="/refguide7/start-event/" >}} | [Start Event](/refguide7/start-event/) | A start event is the starting point of the microflow. A microflow can only have one start event. |
+| {{< figure src="/attachments/refguide7/desktop-modeler/application-logic/microflows/918113.png" link="/refguide7/end-event/" >}} | [End Event](/refguide7/end-event/) | An end event defines the location where the microflow will stop. Depending on the return type of the microflow in some cases a value must be specified. There can be more than one end event. |
+| {{< figure src="/attachments/refguide7/desktop-modeler/application-logic/microflows/16843951.png" >}} link="/refguide7/error-event/" | [Error Event](/refguide7/error-event/) | An error event defines a location where the microflow will stop and throw an error that occurred earlier. If you call a microflow, you may want to know whether any errors occurred within the microflow or not. |
```
Comment (MarkvanMents) [inline content/en/docs/refguide7/desktop-modeler/application-logic/microflows/_index.md]:
```suggestion
| {{< figure src="/attachments/refguide7/desktop-modeler/application-logic/microflows/16843951.png" link="/refguide7/error-event/" >}}  | [Error Event](/refguide7/error-event/) | An error event defines a location where the microflow will stop and throw an error that occurred earlier. If you call a microflow, you may want to know whether any errors occurred within the microflow or not. |
```

File: content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/activities/_index.md [hunk 3]
```diff
@@ -22,90 +22,90 @@ Object activities can be used to create and manipulate objects. The [domain mode
...
+| {{< figure src="/attachments/refguide/modeling/application-logic/microflows-and-nanoflows/activities/commit-object.png" alt="commit object" link="/refguide/committing-objects/" >}} | [Commit object(s)](/refguide/committing-objects/) | Allows you to commit changes to one or more objects. |
+| {{< figure src="/attachments/refguide/modeling/application-logic/microflows-and-nanoflows/activities/create-object.png" alt="create object" link="/refguide/create-object/" >}} | [Create object](/refguide/create-object/) | Creates an object. |
+| {{< figure src="/attachments/refguide/modeling/application-logic/microflows-and-nanoflows/activities/delete-object.png" alt="delete object" link="/refguide/deleting-objects/" >}} | [Delete object(s)](/refguide/deleting-objects/) *(microflows only)* | Deletes an object. |
+| {{< figure src="/attachments/refguide/modeling/application-logic/microflows-and-nanoflows/activities/retrieve.png" alt="retrieve" link="/refguide/retrieve/" >}}](/refguide/retrieve/) | [Retrieve](/refguide/retrieve/) | Gets one (or more) associated objects of another object. Furthermore, this activity can also get one or more objects directly from a database. |
```
Comment (MarkvanMents) [inline content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/activities/_index.md]:
```suggestion
| {{< figure src="/attachments/refguide/modeling/application-logic/microflows-and-nanoflows/activities/retrieve.png" alt="retrieve" link="/refguide/retrieve/" >}} | [Retrieve](/refguide/retrieve/) | Gets one (or more) associated objects of another object. Furthermore, this activity can also get one or more objects directly from a database. |
```

File: content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/nanoflows/_index.md [hunk 4]
```diff
@@ -74,29 +74,29 @@ Events represent the start and endpoints of a nanoflow and special operations in
...
+| {{< figure src="/attachments/refguide/modeling/application-logic/microflows-and-nanoflows/microflows/end-event.png" alt="end event" link="/refguide/end-event/" >}} | [End event](/refguide/end-event/) | Defines the location where the nanoflow will stop. Depending on the return type of the nanoflow, in some cases a value must be specified. There can be more than one end event. |
+| {{< figure src="/attachments/refguide/modeling/application-logic/microflows-and-nanoflows/microflows/error-event.png" link="/refguide/error-event/" >}} | [Error Event](/refguide/error-event/) | An error event defines a location where the nanoflow will stop and throw an error that occurred earlier. If you call a nanoflow, you may want to know whether any errors occurred within the nanoflow or not. |
+| {{< figure src="/attachments/refguide/modeling/application-logic/microflows-and-nanoflows/microflows/continue-event.png" alt="continue event" link="/refguide/continue-event/" >}}](/refguide/continue-event/) | [Continue event](/refguide/continue-event/) | Used to stop the current iteration of a loop and continue with the next iteration. Continue events can only be used inside a [loop](/refguide/loop/). |
```
Comment (MarkvanMents) [inline content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/nanoflows/_index.md]:
```suggestion
| {{< figure src="/attachments/refguide/modeling/application-logic/microflows-and-nanoflows/microflows/continue-event.png" alt="continue event" link="/refguide/continue-event/" >}} | [Continue event](/refguide/continue-event/) | Used to stop the current iteration of a loop and continue with the next iteration. Continue events can only be used inside a [loop](/refguide/loop/). |
```

File: content/en/docs/refguide7/desktop-modeler/application-logic/common-elements/activities/_index.md [hunk 5]
```diff
@@ -72,4 +72,4 @@ See [Microflow Activities](/refguide7/microflow-activities/).
 
 | Graphic | Name | Description |
 | --- | --- | --- |
-| [{{< figure src="/attachments/refguide7/desktop-modeler/application-logic/common-elements/activities/918124.png" >}}](/refguide7/generate-document/) | [Generate document](/refguide7/generate-document/) | Generate document can be used to create a document of a certain type based on a [template](/refguide7/document-templates/). |
+| {{< figure src="/attachments/refguide7/desktop-modeler/application-logic/common-elements/activities/918124.png" link="/refguide7/generate-document/" >}}](/refguide7/generate-document/) | [Generate document](/refguide7/generate-document/) | Generate document can be used to create a document of a certain type based on a [template](/refguide7/document-templates/). |
```
Comment (MarkvanMents) [inline content/en/docs/refguide7/desktop-modeler/application-logic/common-elements/activities/_index.md]:
```suggestion
| {{< figure src="/attachments/refguide7/desktop-modeler/application-logic/common-elements/activities/918124.png" link="/refguide7/generate-document/" >}} | [Generate document](/refguide7/generate-document/) | Generate document can be used to create a document of a certain type based on a [template](/refguide7/document-templates/). |
```

**Review by MarkvanMents — CHANGES_REQUESTED** [review]
@NatasaKralj 
I've made four changes. Can you check that I've understood them correctly.
Didn't realize we used that construction so often! Need to make sure we have this improved format in the documentation.
Thanks for picking this up.
Once you have dealt with the comments, you can merge straight away.

**Pull request merged**